components/support/nimbus-fml/fixtures/nimbus_features.yaml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Could we translate the IR into YAML and put it in a directory in `fixtures`, and then generate the code again and run the same tests that are running off of the IR?

I think `workflows/mod.rs` should be the place to do the running of the tests.